tests/phpunit/CRM/Financial/BAO/PaymentProcessorTest.php

```diff
@@ -62,38 +62,4 @@ public function testGetCreditCards() {
     $this->assertEquals($cards, $expectedCards, 'Verify correct credit card types are returned');
   }
 
-  public function testCreditCardCSSName() {
```

Comment:
function no longer exists to test - it needed to be protected with a test before because it was so obvious a target :-)

Reply:
@eileenmcnaughton i think we should still replace it with some kind of a test because part of this test wasn't just testing the CSS Names but verifying that when a payment processor only accepted specific card types that only those were sent to the form

Reply:
Hmm I guess we could extract 
so we have this test

```
  /**
   * Check method create()
   */
  public function testGetCreditCards() {
    $params = array(
      'name' => 'API_Test_PP_Type',
      'title' => 'API Test Payment Processor Type',
      'class_name' => 'CRM_Core_Payment_APITest',
      'billing_mode' => 'form',
      'payment_processor_type_id' => 1,
      'is_recur' => 0,
      'domain_id' => 1,
      'accepted_credit_cards' => json_encode(array(
        'Visa' => 'Visa',
        'Mastercard' => 'Mastercard',
        'Amex' => 'Amex',
      )),
    );
    $paymentProcessor = CRM_Financial_BAO_PaymentProcessor::create($params);
    $expectedCards = array(
      'Visa' => 'Visa',
      'Mastercard' => 'Mastercard',
      'Amex' => 'Amex',
    );
    $cards = CRM_Financial_BAO_PaymentProcessor::getCreditCards($paymentProcessor->id);
    $this->assertEquals($cards, $expectedCards, 'Verify correct credit card types are returned');
  }
```